torch/_decomp/decompositions.py

```diff
@@ -1200,3 +1203,23 @@ def stack(tensors: List[Tensor], dim: int = 0) -> Tensor:
         return out.view(result_sizes)
     else:
         return torch.cat(get_stack_inputs(tensors, wrapped_dim), dim)
+
+
+def _squeeze_multiple(self, dims):
+    ndims = self.dim()
+    for idx in range(ndims - 1, -1, -1):
```

Comment:
is this correct from dim=-2?

Reply:
I need to add a wrap dim here.

Reply:
A bit surprised it's not tested in OpInfos - added a test there.